core/templates/dev/head/components/forms/HtmlSelectDirective.js

```diff
@@ -33,6 +33,15 @@ oppia.directive('htmlSelect', [function() {
       $scope.select = function(id) {
         $scope.selection = id;
       };
+
+      $scope.getSelectionIndex = function() {
+        for (var i = 0; i < $scope.options.length; i++) {
+          if (($scope.options[i].val === $scope.selection) ||
```

Comment:
NB: I haven't checked carefully, and I'm about to go to bed. So I'm not sure if it's the id or value we should be using (I might have switched them round). But I _think_ one of the fields should work?